### ISSUE TITLE
[Playback speed] <Allow the user to select the video playback speed (e.g. 1.5x/2x) >

### ISSUE THREAD

**Issue by dinojs** [SuperMarcus/NineAnimator#98]
###### This feature is available on few sources supported by the app (e.g. gogoanime)
![image](https://user-images.githubusercontent.com/22294355/79623425-22b77980-8114-11ea-92c5-f7b9a44a810e.png)

**Comment by Catta1997** [comment]
I don't think it will be possible. I think that the official iOS reader does not have this feature
